codice/ddf
# DDF-#05182 Allow client filter to be saved with geo filters

#### What does this PR do?
Allow client filter to be saved with geo filters

#### Who is reviewing it? 
@djblue 
@andrewkfiedler 
@vinamartin 

#### Select relevant component teams: 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.
@djblue 
@andrewkfiedler
@vinamartin 

#### How should this be tested?
Create a search.
Add any geometry based filter on the client side filter. 
Save and reopen the client side filter and ensure the filter just created is still present. 
Refresh the page and repeat. 

#### What are the relevant tickets?
Fixes: #05182

#### Screenshots


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
